website/docs/project_settings/settings_project_global.md

```diff
@@ -263,10 +263,25 @@ Template may look like `"{family}{Task}{Variant}"`.
 Some creators may have other keys as their context may require more information or more specific values. Make sure you've read documentation of host you're using.
 
 
-## Workfiles
+### Publish
+
+#### Custom Staging Dir Profiles
+With this feature, users can specify a transient data folder path based on presets, which can be used during the creation and publishing stages.
+
+![global_tools_custom_staging_dir](assets/global_tools_custom_staging_dir.png)
+
+Staging dirs are used as a destination for intermediate files (as renders) before they are renamed and copied to proper location during integration phase. They could be created completely dynamically in temp folders, or for some DCCs in `work` area.
+(Example could be Nuke where artist might want to temporarily render pictures into `work` area to check them before they get published.)
+One of the key advantages of this feature is that it allows users to repoint folder for such intermediate files to take advantage of faster storages for rendering, which can help improve workflow efficiency. Additionally, this feature allows users to keep their rendered data persistent, and use their own infrastructure for regular cleaning.
+
+In some cases, these DCCs automatically add a rendering path during the creation stage, which is then used in publishing. Creators of such DCCs need to use these profiles to use this functionality.
```

Comment:
I'd argue this is maybe not entirely up to the creators. This might then also need validators, etc. I'd argue that the best case is being able to get the logic working in the Extractor (e.g. choosing the output path during extracting for e.g. a Houdini ROP node and then reverting it.)